README.md

```diff
@@ -23,7 +25,7 @@ A React Native app for streaming WMBR 88.1 FM and browsing show archives, song h
 - Copy `ios/.xcode.env` to `ios/.xcode.env.local` and modify the node path
 - Open the app in Xcode:
   ```bash
-  open ios/WMBRRadioApp.xcworkspace
+  open ios/WmbrApp.xcworkspace
   ```
 - Build the app (⇧⌘B)
 - Run iOS simulator:
```

Comment:
I think this is the same process as Run the Simulator below, but missing your fix for `npm start` - might be worth removing this one/linking to that section?

Reply:
Yes, because README was updated in another PR, so this is targeting a different version than when I wrote it